packages/react-workers/src/utils/sorting.js

```diff
@@ -0,0 +1,66 @@
+import { firstBy } from 'thenby';
+
+/**
+ * Apply sort structure to a collection of features
+ * @param {array} features
+ * @param {object} [sortOptions]
+ * @param {string | string[] | object[]} [sortOptions.sortBy] - One or more columns to sort by
+ * @param {string} [sortOptions.sortByDirection] - Direction by the columns will be sorted
+ */
+export function applySorting(features, { sortBy = [], sortByDirection = 'asc' } = {}) {
+  const isValidSortBy =
+    (Array.isArray(sortBy) && sortBy.length) || // sortBy can be an array of columns
+    typeof sortBy === 'string'; // or just one column
+
+  if (!isValidSortBy) {
+    return features;
```

Comment:
No error here? Shouldn't we just throw with a proper message?

Reply:
Do you think we should throw an error or just show a console.error?

Reply:
Throw in my opinion. Passing an invalid sortby, without noticing, can lead to difficult bugs to find

Reply:
Added